src/app/shared/services/layout.service.ts

```diff
@@ -8,17 +8,17 @@ import { delay } from 'rxjs/operators';
 export class LayoutService {
   // Service allowing modifications of the layout
 
-  private leftMenuAndHeadersDisplayed = new BehaviorSubject<boolean>(true);
-  leftMenuAndHeadersDisplayed$ = this.leftMenuAndHeadersDisplayed.asObservable().pipe(delay(0));
+  private fullFrameContent = new BehaviorSubject<boolean>(true);
+  fullFrameContent$ = this.fullFrameContent.asObservable().pipe(delay(0));
 
-  private withTask = new BehaviorSubject<boolean>(false);
-  withTask$ = this.withTask.asObservable().pipe(delay(0));
+  private footer = new BehaviorSubject<boolean>(false);
+  footer$ = this.footer.asObservable().pipe(delay(0));
```

Comment:
I think it would be useful to add one sentence of comment for `fullFrameContent` and `*footer` so that it is clear what is the intend of these.

Reply:
Added comments on the public observable and function.